libs/cli-kontrol-api/specs/api.yaml

```diff
@@ -5,6 +5,8 @@ info:
 paths:
   /flow/create:
     post:
+      parameters:
```

Comment:
Why not add it as part of the URL path? if the reason is to keep local/dev mode support, can we at least add a TODO to refactor this later?

Reply:
something like `tenant/<UUID>/flow/create`?

Reply:
ok, I'll make the change and add it to the path, I added it as a parameter because I followed the current pattern for namespaces, but I have no objections to putting it into the path.